simulation_parameters/common/difficulty_presets.json

```diff
@@ -23,6 +24,7 @@
     "osmoregulationMultiplier": 1,
     "freeGlucoseCloud": true,
     "passiveReproduction": true,
+    "patchMapExplorationMode": "Intense",
```

Comment:
I think all options should default to the normal exploration mode because I think the going in blind to a patch is much too punishing to increase the fun in the game, even for the hard difficulty. It should be a custom difficulty thing to enable it.